opentelemetry-api/src/opentelemetry/_logs/_internal/__init__.py

```diff
@@ -84,6 +85,7 @@ def __init__(
         severity_number: Optional[SeverityNumber] = None,
         body: AnyValue = None,
         attributes: Optional[_ExtendedAttributes] = None,
+        event_name: Optional[str] = None,
```

Comment:
I think we can remove this since this overload should be deprecated

---

@tammy-baylis-swi it looks like we didn't go ahead with marking the signature as deprecated https://github.com/open-telemetry/opentelemetry-python/pull/4597/commits/dca0d4113b8a690e71cf04a05cc05e91d13bfc1c

Was there an issue with that?